task-launcher/src/tasks/shared/helpers/fetchAndParseCorpus.js

```diff
@@ -82,20 +86,25 @@ const transformCSV = (csvInput, numOfPracticeTrials, sequentialStimulus) => {
     }
 
     // Only push in the specified amount of practice trials
-    if (newRow.notes !== 'practice') {
-      stimulusData.push(newRow);
+    if (newRow.trialType === "instructions") {
+      instructionData.push(newRow);
+      totalInstruction += 1;
     } else if (newRow.notes === 'practice' && currPracticeAmount < numOfPracticeTrials) {
-      stimulusData.push(newRow);
+      practiceData.push(newRow);
       currPracticeAmount += 1;
+      totalPractice += 1;
+    } else if (newRow.notes !== 'practice') {
+      stimulusData.push(newRow);
+      totalStimulus += 1;
     }
   });
 
   // Adjust totalTrials to account for practice trials that might not be added
   totalTrials -= totalPractice - numOfPracticeTrials;
 
-  if (!sequentialStimulus) {
-    stimulusData = shuffleStimulusTrials(stimulusData);
-  }
+  // if (!sequentialStimulus) {
+  //   stimulusData = shuffleStimulusTrials(stimulusData);
+  // }
 
```

Comment:
These changes break other apps and how we're currently sorting. Can you please revert them?